api/tests/authentication_tests.py

```diff
@@ -30,116 +35,121 @@ def test_no_bearer_in_authorization_returns_none(self):
         get_addresses_req = self.factory.get(self.path, **headers)
         assert self.auth.authenticate(self.auth, get_addresses_req) == None
 
-    def test_no_token_returns_none(self):
-        headers = {"HTTP_AUTHORIZATION": "Bearer "}
-        get_addresses_req = self.factory.get(self.path, **headers)
-        with pytest.raises(AuthenticationFailed):
-            self.auth.authenticate(self.auth, get_addresses_req)
+    def test_no_token_returns_400(self):
+        client = APIClient()
+        client.credentials(HTTP_AUTHORIZATION="Bearer ")
+        response = client.get("/api/v1/relayaddresses/")
+        assert response.status_code == 400
```

Comment:
👍 I like testing the API endpoint and status code rather than the authentication method